airflow-core/tests/unit/models/test_dag.py

```diff
@@ -1399,8 +1399,7 @@ def test_create_dagrun_job_id_is_set(self, testing_dag_bundle):
 
     def test_dag_add_task_checks_trigger_rule(self):
         # A non fail stop dag should allow any trigger rule
-        from airflow.exceptions import FailFastDagInvalidTriggerRule
-        from airflow.utils.trigger_rule import TriggerRule
+        from airflow.sdk.exceptions import FailFastDagInvalidTriggerRule
```

Comment:
This isn't correct, we should not be importing from sdk in here.

Reply:
You could move this test to the task sdk where it belongs tbh.

Reply:
Already moved to SDK. And removed from core.